tools/sidekick/config.js

```diff
@@ -41,6 +41,39 @@ function hasSchema(host) {
       },
     ],
     plugins: [
+      {
+        id: 'register-caas',
+        condition: (s) =>
+          s.isHelix() && s.isContent() && !window.location.pathname.endsWith('.json'),
+        button: {
+          text: 'Register with CaaS',
+          action: async (_, sk) => {
+            const SCRIPT_ID = 'publish-caas-listener';
```

Comment:
nit: it might be a little easier to read if you move this function up into it's own named function.